shared/actions/platform-specific/index.native.tsx

```diff
@@ -62,61 +62,42 @@ const requestPermissionsToWrite = async () => {
 
 export const requestAudioPermission = async () => {
   let chargeForward = true
-  // TODO use expo-av etc and unify around that
-  const Permissions = require('expo-permissions') as typeof PermissionsType
```

Comment:
using expo-permissions is deprecated. instead we use the module specific permissions checks